### PR TITLE
docs: ranking points and cosmetic xp docpage

### PR DESCRIPTION
**Pull request by antjlo**
refer to #255

### CODE REVIEW

**Comment by antjlo** [comment]
Was trying to rename it from feat to docs but I suck at git apparently

**Comment by antjlo** [comment]
You can merge it now I renamed it

File: _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+permalink: /guide/ranking-cosmetic-xp/
+
+category: guide
+
+tags:
+  - rank
+  - ranking
+  - xp
+  - experience
+  - exp
```
[the remark this answers is not in the document]
Comment (Gocnak) [reply _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md:13]: XP can be kept along with the full word. The more tags, the easier the search process, but you don't want exp and XP, for example

[the remark this answers is not in the document]
Comment (braem) [reply _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md:13]: Figured searching for XP would have experience come up anyhow, but maybe not always yeah. Exp for instance would have both exp and experience show.
Mainly trying to cut down on the clutter of the tags page, there's a lot of tags

**Comment by antjlo** [comment]
Sorry for the late reply, #git-activity doesn't show this and I haven't had time to check github

File: _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+We choose `A` = 50000, `B` = 49.\
+In this case the graph shows our equation `Points = 50000 / (rank + 49)`\
+![Formula Graph](https://i.imgur.com/wjoJYNg.png)
+
+{:.notice--info}
+X axis designates player rank on a map,\
+y axis designates points awarded
+
+<sub>Ex:\
+Player sets a time of rank 468 on a map with 1333 completions.\
```
Comment (braem) [inline _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md:64]: Note that this renders as
image

Will have to use `<br>` when inside an html tag I believe. There are other instances of this in this guide as well.

File: _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+4 groups, loosely “percentile”-based (in fact, a function of #completions, but not as simple as a percentage). Instead of calculating group boundaries directly from percentiles, we calculate group _sizes_ based on a simple maximizing function --
+
+Group size = `max(A(x), minSize)`
+
+<sup>_max(A, B) simply returns the value of the larger parameter._\
+_A is a function of #completions, minSize_</sup>
+
+Group 1 always starts at rank 11, and its size is determined from its group size formula.
+
+The formula to determine group size, A(x) (where x is the # of completions of the map), is\
```
Comment (braem) [inline _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md:82]:
```suggestion
The formula to determine group size, `A(x)` (where `x` is the # of completions of the map), is\
```

**Review by braem — CHANGES_REQUESTED** [review]
Just a couple more things, mainly that backslashes are being rendered instead of intended newlines

File: _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+## (New) Group System
+
+(based on @Joe’s & Jux’s comments about minimum group sizes)\
+(plus lots of help from Jux and Aux)
+
+4 groups, loosely “percentile”-based (in fact, a function of #completions, but not as simple as a percentage). Instead of calculating group boundaries directly from percentiles, we calculate group _sizes_ based on a simple maximizing function --
+
+Group size = `max(A(x), minSize)`
+
+<sup>_max(A, B) simply returns the value of the larger parameter._\
```
Comment (braem) [inline _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md:77]:
```suggestion
<sup>_max(A, B) simply returns the value of the larger parameter._<br>
```

File: _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+
+minSize is another parameter which varies per group:
+```
+Group 1: minSize = 10
+Group 2: minSize = 45
+Group 3: minSize = 125
+Group 4: minSize = 250
+```
+
+<sup>Ex.\
```
Comment (braem) [inline _posts/guides/2022-2-1-guide_ranking-cosmetic-xp.md:100]:
```suggestion
<sup>Ex.<br>
```

**Review by braem — CHANGES_REQUESTED** [review]

**Comment by antjlo** [comment]
> Feel free to just use github to commit the suggestion - commits will be squashed on merge anyhow

I only realized this yesterday lol, you don't know how long it took for me to realize this

**Review by braem — APPROVED** [review]

**Comment by braem** [comment]
Good to go - just has to be rebased. Odd as there are no actual conflicts but that merge commit is probably what's killing ya.

You can use `git reset HEAD~6` to go back 6 commits, then commit again, making sure it's just the guide. You would need to then force push (`git push --force`)

**Comment by braem** [comment]
> Good to go - just has to be rebased. Odd as there are no actual conflicts but that merge commit is probably what's killing ya.
> 
> You can use `git reset HEAD~6` to go back 6 commits, then commit again, making sure it's just the guide. You would need to then force push (`git push --force`)

Nevermind, squash and rebase was smart enough to resolve

**Pull request merged**